### PR TITLE
moved text right in alert

### PR DESCRIPTION
**Pull request by all-self-taught**
Placed text in red alert and moved to the right.

### CODE REVIEW

File: web/public/templates/dojos/events/user-events.dust [hunk 1]
```diff
@@ -3,42 +3,45 @@
...
-              <td colspan="9">
-                <div class="span9">
-                  <cd-apply-for-event></cd-apply-for-event>
-                </div>
-              </td>
-            </tr>
-            </tbody>
-          </table>
-        </div>
+  <!--<div class="panel panel-body" ng-if="!dojosEvents || dojosEvents.length === 0 || !currentEvents">
```
Comment (Wardormeur) [inline web/public/templates/dojos/events/user-events.dust]: Remove comment? :)

File: web/public/css/_variables.less [hunk 2]
```diff
@@ -30,4 +30,4 @@
 @state-warning-bg: @cd-yellow;
 @state-warning-text: @cd-gray;
 @state-danger-bg: @cd-red;
-@state-danger-text: @cd-red;
+@state-danger-text: @cd-white;
```
Comment (Wardormeur) [inline web/public/css/_variables.less]: That's a bootstrap-wide change, do you see any scenario where we override the background on danger ?@DanielBrierton

Comment (all-self-taught) [inline web/public/css/_variables.less]: I did a grep on the danger alert, and found no other results.

File: web/public/templates/dojos/events/user-events.dust (hunk 1 above)
Comment (all-self-taught) [inline web/public/templates/dojos/events/user-events.dust]: Comment was left in there in case you simply wanted the text moved, as opposed to a red alert.

**Comment by Wardormeur** [comment]
This page has been reworked in the meantime, closing off, sorry @all-self-taught

**Pull request closed without merging**